addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -8026,7 +8029,7 @@ QUnit.module("Views", (hooks) => {
         await clickSave(target);
         assert.containsOnce(
             target,
-            ".alert .o_field_translate",
+            ".o_form_sheet_bg .alert .o_field_translate",
```

Comment:
I don't think this checks order relative to the status bar? You should probably add an assert for that instead of modifying these, the status bar is also in the o_for_sheet_bg.